qiskit_nature/second_q/algorithms/excited_states_solvers/qeom_electronic_ops_builder.py

```diff
@@ -107,9 +107,7 @@ def _build_single_hopping_operator(
         label.append(f"+_{occ}")
     for unocc in excitation[1]:
         label.append(f"-_{unocc}")
-    fer_op = FermionicOp(
-        {" ".join(label): 4.0 ** len(excitation[0])}, num_spin_orbitals=2 * num_spatial_orbitals
-    )
+    fer_op = FermionicOp({" ".join(label): 1.0}, num_spin_orbitals=2 * num_spatial_orbitals)
```

Comment:
Both should work the same, I got used to using the latter because it normalises the S matrix in HX = SXE but I don't mind sticking with the original choice

Reply:
You can leave it as it is now. I just wanted to know :+1: